docs/upgrading_to_project_factory_v1.0.md

```diff
@@ -34,7 +34,7 @@ provider "gsuite" {
 module "project-factory" {
```

Comment:
We shouldn't touch the old upgrading docs as we won't be backporting this change.

Reply:
ah in that way, sure, will roll it back :)